ert3/evaluator/_builder.py

```diff
@@ -106,16 +102,13 @@ def add_step_outputs(
     step: StepBuilder,
 ) -> None:
     for record_name, output in step_config.output.items():
+        transformation = _get_output_recordtransformation(output)
```

Comment:
This might not be part of this PR, but shouldn't we also create `LinkedOutput` class then?

Reply:
The outputs doesn't have the same kind of configurability as the inputs, so it isn't currently needed. The reason linked inputs exist, is due to the separate stages and ensemble configuration scheme, where the same input (record) can serve multiple ensembles. This doesn't currently apply for outputs.